tox.ini

```diff
@@ -38,3 +38,11 @@ skip_install = true
 deps = isort
 commands = isort -c -rc -df spid-testenv.py testenv
 skip_install = true
+
+[flake8]
+# Flake8 can be configured here.
+ignore =
+  E501  # Line too long
+  W503  # line break before binary operator
+exclude =
+  .tox
```

Comment:
Questa esclusione non è necessaria, in quanto flake8 viene invocato esplicitamente su file e directory di progetto.

Reply:
Ok. Valutate voi se passare alla modalità con l'exclude o continuare richiamando i moduli manualmente.

Una volta confermato modifico la PR di conseguenza.

Reply:
direi di continuare richiamando i moduli, come suggeriva @sanjioh